Tasks/AzureRmWebAppDeployment/AzureRmWebAppDeployment-Legacy.ps1

```diff
@@ -0,0 +1,134 @@
+param
```

Comment:
Yes copying the originals to Utility-Legacy.ps1/AzureUtility-Legacy.ps1 would solve the problem. Ultimately the legacy implementation should be deleted in a sprint or two after it's served it's purpose.